### PR TITLE
[AKS] also checking ssh_config_file is a file

### PR DESCRIPTION
**Pull request by rjtsdl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by rjtsdl** [comment]
#4587

**Comment by rjtsdl** [comment]
@yugangw-msft easy fix, feel free to merge it anytime.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def secure_copy(user, host, src, dest, key_filename=None, allow_agent=True):
     ssh = paramiko.SSHClient()
     proxy = None
     ssh_config_file = os.path.expanduser("~/.ssh/config")
-    if os.path.exists(ssh_config_file):
+    if os.path.exists(ssh_config_file) and os.path.isfile(ssh_config_file):
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py]: I don't think you need the old condition of `os.path.exists(ssh_config_file)` now.

**Comment by yugangw-msft** [comment]
@rjtsdl, could you please update the history.rst and setup.py with newer versions?

**Comment by rjtsdl** [comment]
@yugangw-msft Merge it?

**Pull request merged**